docs/UserGuide.md

```diff
@@ -131,6 +131,7 @@ You can then access the food item later on for editing, deleting etc.
 * The quantity field consists of 2 entities - `value` and `unit`. The `value` should come before the `unit`.
   * The `value` is compulsory. It must be a positive number with a maximum of 2 decimal places. The maximum value allowed is 100,000.00.
   * The `unit` is optional. If provided, it must consist of only alphabets. Numbers, spaces and special characters are not allowed. If not provided, the default unit - `unit` - will be given.
+    <br>The maximum length of the quantity length is 20 characters.
```

Comment:
Should it be "The maximum length of the quantity unit"?

Reply:
Should it also be given the same level of bullet as the above?

Reply:
Thank you!